panther_lights/src/controller_node.py

```diff
@@ -21,10 +20,89 @@
 
 
 class PantherAnimation:
+    ANIMATION_DEFAULT_PRIORITY = 3
+    ANIMATION_DEFAULT_TIMEOUT = 120
+
     front: Animation
     rear: Animation
-    interrupting = False
-    repeating = False
+    name: str = 'NAME_NOT_DEFINED'
```

Comment:
```suggestion
    name: str = 'UNDEFINED'
```

Wouldn't it look better like this?